addon/modifiers/key-macro.ts

```diff
@@ -0,0 +1,67 @@
+import Modifier from 'ember-modifier';
```

Comment:
👍 Nice! We wanted to go this route with Octane.

Reply:
😊